src/components/Nav/index.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { Drawer, AppBar, Box, Toolbar, IconButton, Typography, Container, Avatar, Tooltip, Button } from '@mui/material';
+import TextField from '@mui/material/TextField';
+import PersonOutlineIcon from '@mui/icons-material/PersonOutline';
+import logoLightMode from '../../assets/images/fit_trax_app_logo_lightmode.png';
+import { avatarStyle, drawerStyle, iconButtonStyle, navStyle, logoStyle, boxStyle, buttonStyle, drawerPaperStyle } from './styles';
+
+const Nav = () => {
+  const [openDrawer, setOpenDrawer] = React.useState(false);
+  const [username, setUsername] = React.useState('');
+  const [age, setAge] = React.useState('');
+  const [fitness, setFitness] = React.useState('');
+  const [gender, setGender] = React.useState('');
+
+  const toggleDrawer = (open: boolean) => (event: React.KeyboardEvent | React.MouseEvent) => {
+    if (event.type === 'keydown' && ((event as React.KeyboardEvent).key === 'Tab' || (event as React.KeyboardEvent).key === 'Shift')) {
+      return;
+    }
+    setOpenDrawer(open);
+  };
+
+  const handleFormSubmit = () => {
+    /**
+     * @TODO: Waiting for API to submit the form
+     */
+    console.log('form gender :>> ', gender);
```

Comment:
do we still need the console logs?

Reply:
The console logs are there to show the local state till we can intergrade with the aPI and save the data. I also added a TODO comment for better understanding. Happy to remove the console logs though if they cause major issues.